tests/test_loop.py

```diff
@@ -223,7 +223,8 @@ def test_loop_ping_timeout(self):
         res = mqtt_client.loop(timeout=2 * keep_alive_timeout)
```

Comment:
This should make the test case more robust:
```suggestion
        res = mqtt_client.loop(timeout=2 * keep_alive_timeout + recv_timeout)
```

Reply:
Done.